io.github.MakovWait.Godots.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id" : "io.github.MakovWait.Godots",
+    "runtime" : "org.freedesktop.Sdk",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11",
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "command": "godots",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "append-pkg-config-path": "/usr/lib/sdk/dotnet7/lib/pkgconfig"
+    },
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--env=DOTNET_ROOT=/app/lib/dotnet",
+        "--env=DOTNET_CLI_TELEMETRY_OPTOUT=true",
+        "--env=DOTNET_NOLOGO=true"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/openjdk11/install.sh"
+            ]
+        },
+        {
+            "name": "dotnet",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/dotnet7/bin/install-sdk.sh",
+                "cp -r /usr/lib/sdk/dotnet7/lib/packs/ /app/lib/dotnet/"
+            ]
+        },
+        {
+            "name" : "Godots",
+            "builddir" : true,
+            "buildsystem" : "simple",
+            "build-commands": [
+                "unzip Godot_v4.1.1-stable_linux.x86_64.zip",
+                "unzip Godot_v4.1.1-stable_export_templates.tpz",
+                "mkdir ./home",
+                "mkdir -p ./home/.local/share/godot/export_templates/4.1.1.stable",
+                "mv templates/* ./home/.local/share/godot/export_templates/4.1.1.stable",
+                "HOME=/run/build/Godots/home ./Godot_v4.1.1-stable_linux.x86_64 --headless --export-release 'Linux/X11' ./godots-bin",
+                "install -D -m755 ./godots-bin /app/bin/godots-bin",
+                "install -D -m755 ./godots.sh /app/bin/godots",
+                "desktop-file-edit --set-icon=$FLATPAK_ID $FLATPAK_ID.desktop",
+                "install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop",
+                "install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg",
+            ],
+            "sources" : [
+                {
+                    "type": "file",
+                    "url": "https://downloads.tuxfamily.org/godotengine/4.1.1/Godot_v4.1.1-stable_linux.x86_64.zip",
```

Comment:
why not building from source?

that's what `org.godotengine.Godot` does.

Reply:
I left it like that for now since on my machine, it takes quite a bit of time to build. I could update it to build from source though, if that's more desirable.

Reply:
that would provide the aarch64 you excluded.

Reply:
Looking at `exports_presets.cfg` upstream, the project currently doesn't support builds for aarch64 I believe, regardless of what the editor supports. At least, that's if my understanding of how Godot builds work. Should I still build Godot from source?

Reply:
> Should I still build Godot from source?

YES